diff --git a/node_modules/fumadocs-mdx/dist/chunk-GB4W3YCZ.js b/node_modules/fumadocs-mdx/dist/chunk-GB4W3YCZ.js
index 0ece630..a02dda7 100644
--- a/node_modules/fumadocs-mdx/dist/chunk-GB4W3YCZ.js
+++ b/node_modules/fumadocs-mdx/dist/chunk-GB4W3YCZ.js
@@ -13,7 +13,8 @@ ${issues.map((issue) => `  ${issue.path}: ${issue.message}`).join("\n")}`
     this.issues = issues;
   }
   async toStringFormatted() {
-    const picocolors = await import("picocolors");
+    const picocolorsModule = await import("picocolors");
+    const picocolors = picocolorsModule.default || picocolorsModule;
     return [
       picocolors.bold(`[MDX] ${this.title}:`),
       ...this.issues.map(
